test/cypress/support/commands.js

```diff
@@ -0,0 +1,45 @@
+// ***********************************************
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+// Get element by data-test attribute
+Cypress.Commands.add('getByDT', (element) => {
+  return cy.get(`[data-test="${element}"]`)
+})
+
+// NOTE/TODO: We can go a step further and not use UI to do repetitive tasks.
+// https://docs.cypress.io/guides/getting-started/testing-your-app.html#Fully-test-the-login-flow-%E2%80%93-but-only-once
```

Comment:
@taoeffect do you think it's possible to follow this convention from now on (and if yes, how). This is highly valuable to avoid creating users and groups over and over again "manually" (a.k.a by using the UI).

Reply:
It's not that simple since we don't use cookies. It is possible to programmatically generate users, yes, but that's something we can do later if there's a real need for it. Right now this would be "premature optimization", especially as this isn't the current bottleneck on test runtime.

Reply:
okay